pom.xml

```diff
@@ -90,7 +95,7 @@
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
-			<version>1.18.16</version>
+			<version>1.18.20</version>
```

Comment:
This also needs to be upgraded within our maven-compiler-plugin section (line 38).